Description: Use python3 in shebangs
 In Debian python is python2.
Author: Tobias Hansen

--- a/sage/src/bin/math-readline
+++ b/sage/src/bin/math-readline
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 # Cleverly run Mathematica with the benefit of readline, which
 # is something the usual commercial mathematica doesn't provide!
--- a/sage/src/bin/sage-cleaner
+++ b/sage/src/bin/sage-cleaner
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 #*****************************************************************************
 # This is the sage monitor *daemon*, which cleans up after Sage.
--- a/sage/src/bin/sage-coverage
+++ b/sage/src/bin/sage-coverage
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 from __future__ import print_function
 
--- a/sage/src/bin/sage-coverageall
+++ b/sage/src/bin/sage-coverageall
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 import os, sys
 
--- a/sage/src/bin/sage-location
+++ b/sage/src/bin/sage-location
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 from __future__ import print_function
 
--- a/sage/src/bin/sage-num-threads.py
+++ b/sage/src/bin/sage-num-threads.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Determine the number of threads to be used by Sage.
 #
--- a/sage/src/bin/sage-pkg
+++ b/sage/src/bin/sage-pkg
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 from __future__ import print_function
 
--- a/sage/src/bin/sage-pypkg-location
+++ b/sage/src/bin/sage-pypkg-location
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 from __future__ import print_function
 from pkg_resources import Requirement, working_set
 import sys
--- a/sage/src/bin/sage-run
+++ b/sage/src/bin/sage-run
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 import os, sys
 from subprocess import call
--- a/sage/src/bin/sage-sws2rst
+++ b/sage/src/bin/sage-sws2rst
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 r"""
 sage-sws2rst
--- a/sage/src/bin/sage-unzip
+++ b/sage/src/bin/sage-unzip
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # This file is a replacement for the 'unzip' command.
 #
